course-v2/layouts/partials/resource_list_item.html

```diff
@@ -1,31 +1,35 @@
-<div class="resource-list-item container">
+<div class="resource-item resource-list-page">
   <div class="row">
     <div class="d-inline-flex">
-      {{ if not .hideThumbnail }}
-        {{ $thumbnailAttrs := partial "get_resource_thumbnail_attributes.html" (dict "resourceType" .params.resourcetype "fileType" .params.file_type) }}
-        <img
-          class="resource-thumbnail"
-          aria-label="{{- $thumbnailAttrs.ariaLabel -}}"
-          src="{{- $thumbnailAttrs.src -}}"
-        />
-      {{ end }}
-
-      {{ if not .hideDownloadIcon }}
-        {{ $downloadableLink := partial "get_resource_download_link.html" .params }}
-          <span class="pr-2">
-            {{ if $downloadableLink }}
-              <a href="{{- $downloadableLink -}}" aria-label="Download file" target="_blank" download>
-                <img
-                  class="hide-offline resource-download"
-                  src="/static_shared/images/download.svg"
-                  aria-hidden="true"
-                />
-              </a>
+      <div class="resource-thumbnail">
+        {{ $resourceCategory := partial "get_resource_category.html" (dict "resourceType" .params.resourcetype "fileType" .params.file_type) }}
+        <div class="resource-type-thumbnail {{ $resourceCategory }}">
+          <div class="d-flex align-self-center">{{- $resourceCategory -}}</div>
+        </div>  
+        <div class="download-container{{if not .params.file_size}} pb-1{{end}}">
+          {{ if not .hideDownloadIcon }}
+            {{ $downloadableLink := partial "get_resource_download_link.html" .params }}
+            <span class="download-link">
+              {{ if $downloadableLink }}
+                <a href="{{- $downloadableLink -}}" target="_blank" download>
```

Comment:
Putting the anchor tag here results in only the download icon, not the entire button, being clickable. The resource type above and the file size below show an I-beam cursor when you hover over them. The areas to the left and right of the tiny download icon are not even clickable, just the icon itself. Do we want to move this anchor tag to the outside, so that the button as a whole is clickable?

Reply:
I do think that it might not be very user friendly to only have such a small clickable space (download icon). 
But let's get a second opinion from @mbilalmughal on this.

@mbilalmughal do you think we should make the whole icon clickable for download, not just the "small download icon" inside of it?
<img width="62" alt="image" src="https://github.com/mitodl/ocw-hugo-themes/assets/109785089/32ccb9d4-a120-4030-bcd4-5cbc40f97a1c">

Reply:
@ibrahimjaved12 @gumaerc To ensure good UX practice, the "Hit Area" of any CTA (call to action) should be at least the size of the thumb, which is typically around 30 to 50 pixels, making it easy for the user to tap or click.

@ibrahimjaved12 I would say Carey is suggesting the right thing; the entire icon should be clickable, and it would be so nice of you if you could add a little [bounce effect](https://codepen.io/halink0803/pen/qdVXqm) to the download icon when hovering.

Reply:
Thanks @mbilalmughal .
Sure, let's do this.

Reply:
Done. Added the requested changes. Also did a little bit of refactoring. I noticed that `resource_url.html` was used to get the downloadable URL of resource, but we already had it as `$downloadableLink`.

I'm still adding a video demonstration of the animation in the PR but it is ready to be re-reviewed